resmon.lua

```diff
@@ -738,6 +818,11 @@ local function site_comparator_by_etd(left, right)
 end
 
 
+local function site_comparator_by_alpha(left, right)
```

Comment:
These are whitespace changes, not sure how or why they got here. Windows -> Linux things? I didn't create them intentionally.